HELSSA_AGENTS/notification_system/app_code/cores/text_processor.py

```
@@ -0,0 +1,366 @@
+            prompt: Generation prompt
+            content_type: Type of content to generate
+            user_type: Type of user requesting generation
+            max_length: Maximum length of generated content
+            
+        Returns:
+            TextProcessingResult with generated content
+        """
+        start_time = time.time()
+        
+        try:
+            self.logger.info(f"Starting content generation for {APP_NAME}")
```

Ruff (0.12.2)

Undefined name `APP_NAME`

(F821)